exercises/practice/linked-list/.meta/config.json

```diff
@@ -0,0 +1,17 @@
+{
+  "authors": ["jiegillet"],
+  "contributors": [],
+  "files": {
+    "example": [
+      ".meta/example.ex"
+    ],
+    "solution": [
+      "lib/linked_list.ex"
+    ],
+    "test": [
+      "test/linked_list_test.exs"
+    ]
+  },
+  "blurb": "Implement a doubly linked list.",
```

Comment:
```suggestion
  "blurb": "Simulate a doubly linked list.",
```

Reply:
Don't we need to use exactly the same blurb as in https://github.com/exercism/problem-specifications/blob/main/exercises/linked-list/metadata.yml#L2 ?

Configlet will soon have a feature that allows updating blurbs from problem specs. I can't wait to use it to add a lot of missing dots... it would overwrite a custom blurb.

Reply:
Not sure about that tbh